packages/devtools_test/pubspec.yaml

```diff
@@ -0,0 +1,53 @@
+name: devtools_test
+description: A package containing shared test helpers for Dart DevTools tests.
+
+# Note: this version should only be updated by running tools/update_version.dart
```

Comment:
you'll need to add this pubspec path to the update_version.dart script

Reply:
Thanks! Updated `update_version.dart`.